classes/User.php

```diff
@@ -951,7 +951,7 @@ public function loadTrainingManager(): void
             $this->bloodline_id);
     }
 
-    public function setForbiddenSealFromDb(string $forbidden_seal_db, bool $remote_view) {
+    public function setForbiddenSealFromDb(string $forbidden_seal_db, bool $remote_view, bool $set_seal_bonus) {
```

Comment:
An internal flag might be better on this - E.g, the first time we call this, set the bonuses and set `$this->seal_bonuses_applied = true`, the second time, we check the `seal_bonuses_applied` flag and skip applying them (or override the old ones)